test/e2e/service_test.go

```diff
@@ -111,6 +112,7 @@ func (data *TestData) createClusterIPServiceAndBackendPods(t *testing.T, name st
 // Issue: https://github.com/antrea-io/antrea/issues/2289
 func TestNodePortWindows(t *testing.T) {
 	skipIfNoWindowsNodes(t)
+	skipIfNotRequired(t, "service")
```

Comment:
I didn't find where this key word is used?

Reply:
fixed with key word `not-timeconsuming-case`